Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1108,14 +1108,16 @@
 
 
 #if ENABLE(QUICKLOOK_SANDBOX_RESTRICTIONS)
-(with-filter (require-not (state-flag "EnableQuickLookSandboxResources"))
+(allow syscall-unix
+    (syscall-unix-rarely-in-use)
+    (syscall-unix-rarely-in-use-blocked-in-lockdown-mode))
+(with-filter
+    (require-all
+        (state-flag "ParentProcessCanEnableQuickLookStateFlag")
+        (require-not (state-flag "EnableQuickLookSandboxResources")))
```

Comment:
Ah, good point! Looking closer, it seems like the number of brackets are correct, since the last one in the `require-all` condition is matching `(require-all`?

Thanks for reviewing!

Reply:
ok! By the way, the PR contains 2 commits now, so you might want to delete one?